### PR TITLE
[Doc]Refine README in cpp about building

### PR DESCRIPTION
**Pull request by Thespica**
## Proposed changes

Refine build command in README in cpp.

## Types of changes

What types of changes does your code introduce to GraphAr?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/alibaba/GraphAr/blob/main/CONTRIBUTING.rst) doc
- [x] I have signed the CLA
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by acezen — APPROVED** [review]
LGTM

File: cpp/README.rst [hunk 1]
```diff
@@ -101,7 +101,7 @@ After the building, you can install the GraphAr C++ library with:
 
 .. code-block:: shell
 
-    $ sudo make install
+    $ sudo make install  # run in directory you build, like build-relese, build and so on
```
Comment (lixueclaire) [inline cpp/README.rst:104]: build-release

File: cpp/README.rst [hunk 2]
```diff
@@ -57,7 +57,7 @@ Release build:
     $ mkdir build-release
     $ cd build-release
     $ cmake ..
-    $ make -j8       # if you have 8 CPU cores, otherwise adjust
+    $ make -j8       # if you have 8 CPU cores, otherwise adjust, use -j`nproc` for all cores
```
Comment (lixueclaire) [inline cpp/README.rst:60]: Maybe we can unify the usage of ".. code-block:: shell"  ".. code-block::" and ".. code:: shell" in this file

Comment (lixueclaire) [reply cpp/README.rst:60]: > Thanks! I unified it in both README of cpp and README of spark.
> 
> Has any other place where I can make it better?

Thanks for your contribution! This change looks good to me now.

**Review by lixueclaire — APPROVED** [review]

**Pull request merged**